frontend/integration-tests/views/modal-add-secret-to-workload.view.ts

```diff
@@ -0,0 +1,43 @@
+import { $, browser, by, ExpectedConditions as until, element } from 'protractor';
+import * as secretsView from '../views/secrets.view';
+import * as monitoringView from '../views/monitoring.view';
+import { execSync } from 'child_process';
+import * as _ from 'lodash';
+
+
+const selectWorkloadBtn = $('#co-add-secret-to-workload__workload');
```

Comment:
This should be path of `secrets.view.ts, if we go with just one secret scenario file

Reply:
do we also need `modal-add-secret-to-workload.view.ts` added in this PR to `secrets.view.ts`?

Reply:
yes we should